programs/stake/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub fn add_genesis_accounts(genesis_config: &mut GenesisConfig) -> u64 {
 /// NOTE: This is also used to calculate the minimum balance of a stake account, which is the
 /// rent exempt reserve _plus_ the minimum stake delegation.
 #[inline(always)]
-pub(crate) fn get_minimum_delegation(_feature_set: &FeatureSet) -> u64 {
+pub fn get_minimum_delegation(_feature_set: &FeatureSet) -> u64 {
```

Comment:
how about moving this into solana-program, so we don’t need a new dep in ledger/
When it’s moved into solana-program, we could probably remove the FeatureSet as well (since that’s only available in solana-sdk) and instead just always use 1 SOL

Reply:
@mvines Sorry 😢 I already merged this specific part via #24659. I like this suggestion though as it sounds like it'll simplify a number of testing/impl details. So, going forward I could:

1. Revert #24659, then move `get_minimum_delegation()`, then re-do/fix these tests
2. Merge this PR (which will end up not having this specific change since it has already been merged), then move `get_minimum_delegation()`
3. Move `get_minimum_delegation()`, then fixup this PR, then redo #24659 

Do you have a preference here? I feel like (2) is going to be the fastest and most straight forward. Maybe there's another option too?

Reply:
(2), or whatever is easiest wfm!

Reply:
Tracking this in #24690